docs/guides/process-raster-results.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Process raster results
+---
+
+import Image from '../../components/image'
+
+Conveyal provides multiple ways to [compute summary indicators](/analysis/regional#measuring-aggregate-accessibility) for completed regional analyses. Examples include:
+* Average number of schools reachable within a 20 minute bike ride for residents of a city,
+* Number of jobs reachable in a 45 minute commute by the median household in a specific neighborhood, and
+* Increase in these numbers (relative to a baseline scenario) resulting from a proposed transport network.
+
+You may also want compute a single composite index, or a complex indicator combining different logical conditions or spatial layers. The Conveyal team is considering ways to support additional holistic indicators directly in the interface; contact yoru support team if you have specific requests or ideas. In the meantime, you can compute a wider range of indicators by downloading results and running custom calculations in offline GIS software (we recommend QGIS). By default, travel time results, regional access results, and uploaded spatial datasets are all available for download as grid (raster) files with consistent resolutions. This guide provides a few examples of ways to use these raster files.
```

Comment:
Would the sentence starting with "The Conveyal team..." be better as a footnote or call out?